lib/src/components/NxList/NxListLink.tsx

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { forwardRef } from 'react';
+import { faAngleRight } from '@fortawesome/free-solid-svg-icons';
+import { NxFontAwesomeIcon } from '../..';
+import {NxListLinkProps, nxListLinkPropTypes} from './types';
+import classnames from 'classnames';
+import { includesDisabledClass } from '../../util/classUtil';
+
+const NxListLink = forwardRef<HTMLAnchorElement, NxListLinkProps>(
+    function nxListLink({ children, className, disabled, href, selected, ...attrs}, ref) {
+      const aClassNames = classnames('nx-list__link', className, {'selected': selected}, {'disabled': disabled});
```

Comment:
This has the same concerns I mentioned on the button component, that it needs to be clear which element the external `className` prop is getting set on, and that we might want a way to specify extra classes and attrs on both the `li` and the `a`.

Reply:
> This has the same concerns I mentioned on the button component, that it needs to be clear which element the external `className` prop is getting set on, and that we might want a way to specify extra classes and attrs on both the `li` and the `a`.

Implemented for both the button and link components.
1248b66c8c8a03b10c3f514f9af12230277cedb5